poetry/utils/exporter.py

```diff
@@ -81,6 +81,9 @@ def _export_requirements_txt(
                 line = "-e git+{}@{}#egg={}".format(
                     package.source_url, package.source_reference, package.name
                 )
+                if package.source_subdirectory:
+                    line += "?subdirectory={}".format(package.source_subdirectory)
```

Comment:
```suggestion
                    line += "&subdirectory={}".format(package.source_subdirectory)
```

See https://pip.readthedocs.io/en/stable/reference/pip_install/#vcs-support

Reply:
Hey @abn,
thanks a lot for your comments. You're right here :+1: